lib/elixir/lib/stream.ex

```diff
@@ -423,6 +423,10 @@ defmodule Stream do
   This operation will block the caller by the given interval
   every time a new item is streamed.
 
+  Do not use this function to generate number sequences. If
+  blocking the caller process is not necessary, instead use
+  `Stream.iterate(0, & &1 + 1)`.
```

Comment:
I find it much more readable without the parenthesis. Maybe we should make `Macro.to_string/1` smart enough to handle those cases. It has a tendency to add too many parens. :)